flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -1557,8 +1557,7 @@ public void run() {
 				}
 			}
 			catch (Throwable t) {
```

Comment:
Is it even necessary to catch `Throwable` and rethrow it as a `FlinkRuntimeException`? We are already handling all checked exceptions. Therefore the code would compile without the `try-catch` block and even do something similar.